frontend/src/tests/sightings/Sightings.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from "@testing-library/react";
+import { createMemoryHistory } from "history";
+import { Router } from "react-router-dom";
+import { Sightings } from "../../components/sightings/sightings";
+
+test("Should render without error", () => {
+  const history = createMemoryHistory();
+  render(
+    <Router history={history}>
+      <Sightings />
+    </Router>
+  );
+
+  const elements = screen.getAllByText(/Sighting page!/i);
+  expect(elements[0]).toBeInTheDocument();
+});
+
+describe("Sighting page", () => {
+  it("Should render without error", () => {
+    render(<Sightings />);
+  });
+});
```

Comment:
These have the same name, and should probably be added to the same test suite? I can clarify what I mean!